tests/ert_tests/ert3/engine/integration/test_engine.py

```diff
@@ -238,11 +253,15 @@ def test_run_once_polynomial_evaluation(
             ert3.engine.run(experiment_run_config, workspace, "evaluation")
 
 
+def create_ensemble_file(experiment_file):
+    experiment_file.parent.mkdir(parents=True, exist_ok=True)
```

Comment:
+1

Reply:
This one is actually fine as it can be called a few times :)